components/Editor/index.tsx

```diff
@@ -88,17 +88,16 @@ const Editor = ({ readOnly = false }: Props) => {
 
   const getCallValue = useCallback(() => {
     const _callValue = BigInt(callValue)
-    let cv = 0n
-
-    if (unit === ValueUnit.Gwei) {
-      cv = _callValue * BigInt('1000000000')
-    } else if (unit === ValueUnit.Finney) {
-      cv = _callValue * BigInt('1000000000000000')
-    } else if (unit === ValueUnit.Ether) {
-      cv = _callValue * BigInt('1000000000000000000')
+    switch (unit) {
+      case ValueUnit.Gwei:
+        return _callValue * BigInt('1000000000')
```

Comment:
can we use JS's `_` to help readability: `1_000_000_000` or else powers of 10 notation

Reply:
Yes we can but this is the code from evm.codes I’d leave as is to avoid conflicts on futures catch ups with main codebase

Reply:
Discussed on Telegram @Eikix is ok